GCTC-NTGC/gc-digital-talent
# Update job poster title

Resolves #4574.

Updates the title on the job poster page to use the `formattedPoolPosterTitle` function.

To test:
Navigate to a job posting (`/browse/pools/{id}`) and confirm the title displays in the expected format. Ex: `Pool name ( EX-01 Pool Stream )`
